e2e/replay/msg_persister.go

```diff
@@ -44,25 +44,25 @@ func (r *replayMessagePersister) saveMetaData(nodeNames *[]string) error {
 	return err
 }
 
-// saveCachedMessages saves currently cached messages and timeouts to .flow file
-func (r *replayMessagePersister) saveCachedMessages() error {
+// SaveCachedMessages saves currently cached messages and timeouts to .flow file
+func (r *MessagePersister) SaveCachedMessages() error {
 	r.lock.Lock()
 	defer r.lock.Unlock()
 
 	var err error
-	if err = r.createFile(); err != nil {
+	if err = r.CreateFile(); err != nil {
 		return err
 	}
 
 	if r.messages != nil {
-		err = r.saveMessages(r.file)
+		err = r.SaveMessages(r.file)
 	}
 
 	return err
 }
 
-// createFile creates a .flow file to save messages and timeouts on the predifined location
-func (r *replayMessagePersister) createFile() error {
+// CreateFile creates a .flow file to save messages and timeouts on the predifined location
+func (r *MessagePersister) CreateFile() error {
```

Comment:
Seems like this doesn't need to be public eventually (it is called by `MessageNotifier`, which is in the `replay` package as well)

Reply:
Done